validator/validator-main.protoascii

```diff
@@ -313,6 +313,35 @@ tags: {
   attr_lists: "common-link-attrs"
   spec_url: "https://amp.dev/documentation/guides-and-tutorials/learn/spec/amphtml/#html-tags"
 }
+# Allowlisted CSS provider
+tags: {
+  html_format: AMP
+  enabled_by: "transformed"
+  tag_name: "LINK"
+  spec_name: "link rel=stylesheet for amp-story-1.0 css"
+  descriptive_name: "link rel=stylesheet for amp-story-1.0 css"
+  mandatory_parent: "HEAD"
+  attr_lists: "nonce-attr"
```

Comment:
made sure to not allow `async` here

Reply:
The `link` tag doesn't support `async` anyway?

Reply:
good point. im not sure it is listed below for the font stylesheet